README.md

```diff
@@ -7,3 +7,27 @@ This is a suitcase subpackage for reading a particular file format.
 ```
 pip install suitcase-tiff
 ```
+
+## Quick Start
+
+`suitcase-tiff` supports stack and series.
+
+Using stack:
+
+```
+import suitcase.tiff_stack
+docs = db[-1].documents(fill=True)
+suitcase.tiff_stack.export(, 'my_exported_files/', file_prefix='PREFIX-')
```

Comment:
Missing the first argument, which should be `docs`

Reply:
thanks